Automattic/jetpack
# Protect: Update support links

## Description
Conditionally redirect users to support resources based on upgrade.

### Other information:


## Jetpack product discussion


- https://github.com/Automattic/jetpack-scan-team/issues/1287

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Checkout this branch
* Start Jurassic Tube and install Protect
* Verify that the updated support redirect you appropriately according to your upgrade

## Screenshots

automaticRulesInstallationError:

![Screen Shot on 2024-07-18 at 10-33-16 (1)](https://github.com/user-attachments/assets/1e383ff0-a94a-497a-ac8e-9dc73b88002c)

handleApiError, handleShareDataChange, handleShareDebugDataChange :

![Screen Shot on 2024-07-18 at 10-41-28](https://github.com/user-attachments/assets/b1f97b89-9265-4f60-a024-2bda51e13811)
